usmqe_tests/api/gluster/test_volume.py

```diff
@@ -434,11 +434,3 @@ def test_delete_volume_valid(
             """
     storage = gluster.GlusterCommon()
     storage.find_volume_name(valid_volume_name, False)
-
-    # There should be either deleted attribute or record should be removed from database
-    # https://github.com/Tendrl/api/issues/78
-    #
-    # deleted = api.get_volume_list(valid_cluster_id)[0][valid_volume_id]["deleted"]
```

Comment:
I see that original code there (in ec364f242eb0fe92f6bd97497f17ed54b46e9399) was:

```
value = api.get_volume_attribute(cluster_id, volume_id, "deleted")
pytest.check(value == "True")
```

which was later (in f76222cc38ccbd93222213a135065599c6796272) changed to:

```
deleted = api.get_volume_list(valid_cluster_id)[0][valid_volume_id]["deleted"]
pytest.check(deleted == "True", issue="https://github.com/Tendrl/tendrl-api/issues/33")
```

And then was just commented out in 6e5db512ad651b2b8a21172546d0f25ef289701f.

Since you say that this both issues are fixed and in the test case in question (`test_delete_volume_valid`) you ask tendrl api to remove the volume and then check that via usmqe gluster module, would it make sense to check that the gluster volume was removed from tendrl point of view as well? Should we check that  `api.get_volume_list` no longer contains just deleted volume?